server/test/e2e/formats.e2e-docker-spec.ts

```diff
@@ -0,0 +1,199 @@
+import { AssetResponseDto, ExifResponseDto, LoginResponseDto } from '@app/domain';
+import { AssetType, LibraryType } from '@app/infra/entities';
+import { INestApplication } from '@nestjs/common';
+import { api } from '@test/api';
+import { TEST_ASSET_PATH, createTestApp, db, ensureTestAssets } from '@test/test-utils';
+
+describe(`Supported file formats (e2e)`, () => {
+  let app: INestApplication;
+  let server: any;
+  let admin: LoginResponseDto;
+
+  type Format = [string, string, Partial<AssetResponseDto>, Partial<ExifResponseDto>];
```

Comment:
Maybe we can use something like this instead?
```typescript
interface FormatTest {
  format: string;
  folder: string;
  response: Partial<AssetResponseDto>;
  exifInfo: Partial<ExifResponseDto>;
  }
  ```
  
  and then used in the test as
```typescript
  it.each(tests, ...({ name, extension, response, exifInfo }: FormatTest)
 ```

Reply:
great suggestion, done